src/ImageRunModal.jsx

```diff
@@ -824,222 +824,224 @@ export class ImageRunModal extends React.Component {
+                                        <FormGroup fieldId='run-image-dialog-restart-policy' label={_("Restart policy")}
+                                            labelHelp={
+                                                <Popover aria-label={_("Restart policy help")}
+                                                    enableFlip
+                                                    bodyContent={userLingeringEnabled ? _("Restart policy to follow when containers exit. Using linger for auto-starting containers may not work in some circumstances, such as when ecryptfs, systemd-homed, NFS, or 2FA are used on a user account.") : _("Restart policy to follow when containers exit.")}>
+                                                    <Button variant="plain" hasNoPadding aria-label="More info" icon={<OutlinedQuestionCircleIcon />} />
+                                                </Popover>
+                                            }
+                                        >
+                                            <FormSelect id="run-image-dialog-restart-policy"
+                                                aria-label={_("Restart policy help")}
+                                                value={dialogValues.restartPolicy}
+                                                onChange={(_event, value) => this.onValueChanged('restartPolicy', value)}>
+                                                <FormSelectOption value='no' key='no' label={_("No")} />
+                                                <FormSelectOption value='on-failure' key='on-failure' label={_("On failure")} />
+                                                <FormSelectOption value='always' key='always' label={_("Always")} />
+                                            </FormSelect>
+                                        </FormGroup>
+                                    </GridItem>
+                                    {dialogValues.restartPolicy === "on-failure" &&
+                                        <FormGroup fieldId='run-image-dialog-restart-retries'
+                                            label={_("Maximum retries")}>
+                                            <NumberInput
+                                                id="run-image-dialog-restart-retries"
+                                                value={dialogValues.restartTries}
+                                                min={1}
+                                                max={65535}
+                                                widthChars={5}
+                                                minusBtnAriaLabel={_("Decrease maximum retries")}
+                                                plusBtnAriaLabel={_("Increase maximum retries")}
+                                                onMinus={() => this.onMinusOne('restartTries')}
+                                                onPlus={() => this.onPlusOne('restartTries')}
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2246-4debadf1-20250829-124849-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)